wm/border.go

```diff
@@ -1,19 +1,42 @@
 package wm
 
 import (
+	"io/ioutil"
+	"log"
+	"path/filepath"
+
+	"github.com/fyne-io/desktop"
+
 	"fyne.io/fyne"
+
 	"fyne.io/fyne/canvas"
 	"fyne.io/fyne/layout"
 	"fyne.io/fyne/theme"
 	"fyne.io/fyne/widget"
 )
 
-func newBorder(title string) fyne.CanvasObject {
+func newBorder(title string, class []string, command string, iconName string) fyne.CanvasObject {
```

Comment:
To save exploding this list of parameters can we not pass a desktop.Window?

Reply:
No - I don't think I have a reference to a window at the time newBorder is being called in wm/frame.go?

Reply:
Yes we do. newBorder is called from *frame, which is itself a Window...